### PR TITLE
Feature/6 privacy policy

### PR DESCRIPTION
**Pull request by carlyspikes**
privacy policy page link is in the footer

### CODE REVIEW

**Review by KaungHT13 — CHANGES_REQUESTED** [review]
There were some duplicates. Also, the main website that the users will access will be [https://ahcer.org/](https://ahcer.org/) instead of [https://ahcer.azurewebsites.net/](https://ahcer.azurewebsites.net/), and login route should not be provided for the privacy policy. Apart from those, looks good to me.

File: ahcer/src/app/privacy-policy/privacy-policy.component.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<div class="container">
+<h1>Privacy Policy for AHC</h1>
+
+<p>At e-AHC, accessible from https://ahcer.azurewebsites.net/login, one of our main priorities is the privacy of our visitors. This Privacy Policy document contains types of information that is collected and recorded by e-AHC and how we use it.</p>
```
[the remark this answers is not in the document]
Comment (runrobrun) [reply ahcer/src/app/privacy-policy/privacy-policy.component.html:4]: I would stick with capital E

File: ahcer/src/index.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/quill/1.3.6/quill.snow.min.css">
 </head>
-<body>
+<body class="mat-typography">
```
[the remark this answers is not in the document]
Comment (runrobrun) [reply ahcer/src/index.html:21]: ensures that the site consistently uses same fonts, sizes etc.

**Comment by chimezieokeke** [comment]
In terms of the privacy policy, I checked out your branch and it looks good!

**Review by chimezieokeke — COMMENTED** [review]
Looks good!

File: ahcer/src/app/privacy-policy/privacy-policy.component.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <div class="container">
 <h1>Privacy Policy for AHC</h1>
 
-<p>At e-AHC, accessible from https://ahcer.azurewebsites.net/login, one of our main priorities is the privacy of our visitors. This Privacy Policy document contains types of information that is collected and recorded by e-AHC and how we use it.</p>
+<p>At E-AHC, accessible from https://ahcer.org/, one of our main priorities is the privacy of our visitors. This Privacy Policy document contains types of information that is collected and recorded by e-AHC and how we use it.</p>
```
Comment (KaungHT13) [inline ahcer/src/app/privacy-policy/privacy-policy.component.html:4]: You missed an e-AHC near the end of this line. Also, the same thing at line 8.

File: ahcer/src/app/privacy-policy/privacy-policy.component.html [hunk 4]
```diff
@@ -33,27 +33,27 @@ <h2>How we use your information</h2>
...
 <h2>Advertising Partners Privacy Policies</h2>
 
-<p>You may consult this list to find the Privacy Policy for each of the advertising partners of e-AHC.</p>
+<p>You may consult this list to find the Privacy Policy for each of the advertising partners of E-AHC.</p>
 
-<p>Third-party ad servers or ad networks uses technologies like cookies, JavaScript, or Web Beacons that are used in their respective advertisements and links that appear on e-AHC, which are sent directly to users' browser. They automatically receive your IP address when this occurs. These technologies are used to measure the effectiveness of their advertising campaigns and/or to personalize the advertising content that you see on websites that you visit.</p>
+<p>Third-party ad servers or ad networks uses technologies like cookies, JavaScript, or Web Beacons that are used in their respective advertisements and links that appear on EF-AHC, which are sent directly to users' browser. They automatically receive your IP address when this occurs. These technologies are used to measure the effectiveness of their advertising campaigns and/or to personalize the advertising content that you see on websites that you visit.</p>
```
Comment (KaungHT13) [inline ahcer/src/app/privacy-policy/privacy-policy.component.html:50]: There is an extra F. Should be E-AHC instead of EF-AHC.

**Review by KaungHT13 — APPROVED** [review]
Looks good!

**Pull request merged**